pcsx2/Counters.cpp

```diff
@@ -34,7 +34,6 @@
 using namespace Threading;
 
 extern u8 psxhblankgate;
-extern bool gsIsInterlaced;
```

Comment:
> I'm assuming that now won't compile. (what did I tell you about testing? :P )

Appveyor/TravisCI seems to be happy. :)

> Also below where you removed it from GS.h you will need to move that somewhere relevant as well.

I didn't remove anything from ``GS.h``. I assume you mean the ``gsIsInterlaced`` variable  ? It has been moved to get initialized at the SetGsCrt function.